### PR TITLE
Add wrapping div with search role to search form

### PR DESCRIPTION
**Pull request by Rleahy22**
This was only a 2 line change and the rest is indentation: https://github.com/department-of-veterans-affairs/vets-website/pull/17919/files?diff=split&w=1

## Description
Add `role="search"` to SearchBar so that the search form gets included in landmarks for screen readers

## Original issue(s)
department-of-veterans-affairs/va.gov-cms#13138

## Testing done
Manual

## Screenshots
Screen Shot 2021-07-19 at 4 22 38 PM

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/resources-and-support/tests/components/ResourcesAndSupportSearchApp.unit.spec.jsx [hunk 1]
```diff
@@ -32,6 +32,13 @@ describe('ResourcesAndSupportSearchApp', () => {
     ReactDOM.render(<ResourcesAndSupportSearchApp />, div);
   });
 
+  it('creates a landmark for the search form', async () => {
+    const screen = renderInReduxProvider(<ResourcesAndSupportSearchApp />);
+    await screen.findByLabelText('Enter a keyword, phrase, or question');
+
+    screen.getByRole('search');
+  });
+
```
Comment (zacharymorel) [inline src/applications/resources-and-support/tests/components/ResourcesAndSupportSearchApp.unit.spec.jsx:41]: Thanks for adding test coverage!

**Review by zacharymorel — APPROVED** [review]

File: src/applications/resources-and-support/components/SearchBar.jsx [hunk 2]
```diff
@@ -86,76 +86,81 @@ export default function SearchBar({
...
-                    Search
-                  </span>{' '}
-                  All VA.gov
-                </label>
+              <div className="vads-u-flex--auto vads-u-width--full vads-u-margin-top--2 medium-screen:vads-u-margin-top--0 medium-screen:vads-u-width--auto">
+                <button
+                  className="usa-button vads-u-margin--0 vads-u-width--full vads-u-height--full medium-screen-va-border-left-radius--0"
+                  type="submit"
+                >
+                  <i className="fa fa-search"> </i> Search
```
[the remark this answers is not in the document]
Comment (jhonnyoddball) [reply src/applications/resources-and-support/components/SearchBar.jsx:160]: If your icons are purely decorative, you’ll need to manually add an `aria-hidden` attribute to each of your icons so they’re accessible.

File: src/applications/resources-and-support/components/SearchBar.jsx [hunk 3]
```diff
@@ -86,76 +86,81 @@ export default function SearchBar({
...
-                    Search
-                  </span>{' '}
-                  All VA.gov
-                </label>
+              <div className="vads-u-flex--auto vads-u-width--full vads-u-margin-top--2 medium-screen:vads-u-margin-top--0 medium-screen:vads-u-width--auto">
+                <button
+                  className="usa-button vads-u-margin--0 vads-u-width--full vads-u-height--full medium-screen-va-border-left-radius--0"
+                  type="submit"
+                >
+                  <i className="fa fa-search" aria-hidden /> Search
```
[the remark this answers is not in the document]
Comment (jhonnyoddball) [reply src/applications/resources-and-support/components/SearchBar.jsx:160]: it should be `aria-hidden="true"`

**Review by jhonny218 — APPROVED** [review]
LGTM

**Review by kelsonic — APPROVED** [review]
Nice work! 🎉

**Pull request merged**